api/client/proto/authservice.proto

```diff
@@ -254,6 +254,30 @@ message PingResponse {
     string ClusterName = 1 [ (gogoproto.jsontag) = "cluster_name" ];
     // ServerVersion is the version of the auth server.
     string ServerVersion = 2 [ (gogoproto.jsontag) = "server_version" ];
+    // ServerFeatures are the features supported by the auth server.
+    Features ServerFeatures = 3 [ (gogoproto.jsontag) = "server_features" ];
+    // PublicProxyAddr is the server's public proxy address.
+    string PublicProxyAddr = 4 [ (gogoproto.jsontag) = "public_proxy_addr" ];
```

Comment:
Minor, but we use `public_addr` everywhere, so this should be `proxy_public_addr` and `ProxyPublicAddr`.